src/main/java/org/prebid/server/bidder/BidderErrorNotifier.java

```diff
@@ -69,7 +69,7 @@ private Void handleTimeoutNotificationSuccess(HttpClientResponse response, HttpR
                     String.format(
                             "Notified bidder about timeout. Status code: %s. Request body: %s",
                             response.getStatusCode(),
-                            timeoutNotification.getBody()),
+                            new String(timeoutNotification.getBody())),
```

Comment:
Why we need this?

Reply:
<img width="492" alt="Screenshot 2021-11-09 at 11 13 18" src="https://user-images.githubusercontent.com/56919177/140895900-e5717443-451d-4c4d-9c83-26c6b5f0b413.png">

Reply:
Agreed.